pkg/storage/aws/cloudfront/storage.go

```diff
@@ -19,19 +19,25 @@ type Storage struct {
 
 // Get takes in the Context and path as an argument and returns an IResponse interface implementation.
 // This method figures out how to get the data from the cloudfront storage backend.
-func (s *Storage) Get(ctx context.Context, path string) storage.IResponse {
-	res, err := s.client.Get(s.getURL(path), nil)
+func (s *Storage) Get(ctx context.Context, path string, opt *storage.GetRequestOptions) storage.IResponse {
```

Comment:
Same here. This function should not change. There should be a separate function something like `GetPartialObject`, which accepts the options that are specific to byte range